apollo-federation/src/sources/connect/validation/connect/entity.rs

```diff
@@ -106,13 +115,8 @@ pub(super) fn validate_entity_arg(
         return Err(Message {
             code: Code::EntityResolverArgumentMismatch,
             message: format!(
-                "{coordinate} must have arguments. See https://go.apollo.dev/connectors/directives/#rules-for-entity-true",
-                coordinate = field_with_connect_directive_entity_true_coordinate(
-                    connect_directive_name,
-                    entity_arg_value.as_ref(),
-                    object,
-                    &field.name,
-                ),
+                "`{coordinate}` must have arguments when using `entity: true`. See https://go.apollo.dev/connectors/directives/#rules-for-entity-true",
```

Comment:
yeah, maybe we can just hardcode `entity: true` in the argument-related error messages since the validations only matter when it's `true`. (we'll have separate validations for the `$batch`/`$this` variables for connect on types.